src/annotator/sidebar.js

```diff
@@ -18,8 +18,16 @@ import BucketBar from './plugin/bucket-bar';
  * @prop {number} height
  */
 
+/**
+ * @typedef {{
+ *    eventTarget: Window| HTMLElement,
+ *    eventType: string,
+ *    listener: (event: any) => void;
+ * }} RegisteredListener
+ */
+
 // Minimum width to which the frame can be resized.
-const MIN_RESIZE = 280;
+export const MIN_RESIZE = 280;
```

Comment:
It looks like the reason for exporting this might be a test seam? If so, there might be other ways to provide the test affordance without having to export the constant. Don't tie yourself in knots if you can't find a way to do it within a few minutes...

Reply:
Yes, testing is the only reason why I import the value.
I could create a static member of the class, like this `Sidebar.MIN_SIZE`, but that would make it more difficult, if in the future, we want to refactor the class as a function component.

I would keep it as it is. If you have other suggestion, please let me know.